From 9193db67743d08f418235fee43747bb9d42d62ce Mon Sep 17 00:00:00 2001
From: Michael Jeanson <mjeanson@efficios.com>
Date: Fri, 15 Mar 2019 11:19:27 -0400
Subject: [PATCH] fix: tests: link libpause_consumer on liblttng-ctl

This preload test library uses symbols from liblttng-ctl which are
resolved when preloaded by GLIBC but not by MUSL.

Signed-off-by: Michael Jeanson <mjeanson@efficios.com>
---
 tests/regression/tools/notification/Makefile.am | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/tests/regression/tools/notification/Makefile.am b/tests/regression/tools/notification/Makefile.am
index 33e95886d..b869bdb1d 100644
--- a/tests/regression/tools/notification/Makefile.am
+++ b/tests/regression/tools/notification/Makefile.am
@@ -20,7 +20,10 @@ FORCE_SHARED_LIB_OPTIONS = -module -shared -avoid-version \
 			   -rpath $(abs_builddir)
 
 libpause_consumer_la_SOURCES = consumer_testpoints.c
-libpause_consumer_la_LIBADD = $(top_builddir)/src/common/libcommon.la $(DL_LIBS)
+libpause_consumer_la_LIBADD = \
+	$(top_builddir)/src/common/libcommon.la \
+	$(top_builddir)/src/lib/lttng-ctl/liblttng-ctl.la \
+	$(DL_LIBS)
 libpause_consumer_la_LDFLAGS = $(FORCE_SHARED_LIB_OPTIONS)
 noinst_LTLIBRARIES = libpause_consumer.la
 
